src/v0/destinations/adobe_analytics/transform.js

```diff
@@ -306,7 +306,7 @@ const processTrackEvent = (message, adobeEventName, destinationConfig, extras =
     destinationConfig;
   const { event: rawMessageEvent, properties } = message;
   const { overrideEventString, overrideProductString } = properties;
-  const event = rawMessageEvent.toLowerCase();
+  const event = rawMessageEvent?.toString()?.toLowerCase();
```

Comment:
can we have util to check for 
- event exists ?
- event exists & not object
- event is string & then lowercase conversion